must-call-checker/src/main/java/org/checkerframework/checker/mustcall/MustCallAnnotatedTypeFactory.java

```diff
@@ -67,8 +67,8 @@ public MustCallAnnotatedTypeFactory(final BaseTypeChecker checker) {
     TOP = AnnotationBuilder.fromClass(elements, MustCallUnknown.class);
     BOTTOM = createMustCall();
     POLY = AnnotationBuilder.fromClass(elements, PolyMustCall.class);
-    addAliasedAnnotation(InheritableMustCall.class, MustCall.class, true);
-    addAliasedAnnotation(MustCallChoice.class, POLY);
+    addAliasedTypeAnnotation(InheritableMustCall.class, MustCall.class, true);
```

Comment:
Let's not include this unrelated change.  Probably @kelloggm has fixed on his branch

Reply:
https://github.com/kelloggm/object-construction-checker/pull/300/commits/a71831579d604bf70bc86fe342299488b4f80141